stable/drupal/templates/NOTES.txt

```diff
@@ -1,6 +1,5 @@
-*******************************************************************
-*** PLEASE BE PATIENT: Drupal may take a few minutes to install ***
-*******************************************************************
+
+PLEASE BE PATIENT: Drupal may take a few minutes to install
```

Comment:
We should keep the asterisks in this one. It's only when we're using this with templating that it gets unaligned and looks weird.